marray/tests/test_marray.py

```diff
@@ -59,49 +60,155 @@ def assert_equal(res, ref, seed):
     except AssertionError as e:
         raise AssertionError(seed) from e
 
+
 arithmetic_unary = [lambda x: +x, lambda x: -x, abs]
-arithmetic_binary = [lambda x, y: x + y, lambda x, y: x - y, lambda x, y: x * y,
-                     lambda x, y: x / y, lambda x, y: x // y, lambda x, y: x % y,
-                     lambda x, y: x ** y]
+arithmetic_binary = [
+    lambda x, y: x + y,
+    lambda x, y: x - y,
+    lambda x, y: x * y,
+    lambda x, y: x / y,
+    lambda x, y: x // y,
+    lambda x, y: x % y,
+    lambda x, y: x**y,
+]
 # array operators
 bitwise_unary = [lambda x: ~x]
-bitwise_binary = [lambda x, y: x & y, lambda x, y: x | y, lambda x, y: x ^ y,
-                  lambda x, y: x << y, lambda x, y: x >> y]
-comparison_binary = [lambda x, y: x < y, lambda x, y: x <= y, lambda x, y: x > y,
-                     lambda x, y: x >= y, lambda x, y: x == y , lambda x, y: x != y]
-
-def iadd(x, y): x += y
-def isub(x, y): x -= y
-def imul(x, y): x *= y
-def itruediv(x, y): x /= y
-def ifloordiv(x, y): x //= y
-def ipow(x, y): x **= y
-def imod(x, y): x %= y
+bitwise_binary = [
+    lambda x, y: x & y,
+    lambda x, y: x | y,
+    lambda x, y: x ^ y,
+    lambda x, y: x << y,
+    lambda x, y: x >> y,
+]
+comparison_binary = [
+    lambda x, y: x < y,
+    lambda x, y: x <= y,
+    lambda x, y: x > y,
+    lambda x, y: x >= y,
+    lambda x, y: x == y,
+    lambda x, y: x != y,
+]
+
+
+def iadd(x, y):
+    x += y
+
+
```

Comment:
And these one-liners just don't need four lines, IMO.

Reply:
Maybe. I think I had a bug with the in-place operators anyway, so I'll investigate.